docs/docsite/rst/community/steering/steering_committee_membership.rst

```diff
@@ -48,8 +48,8 @@ Process
 The process to join the Steering Committee consists of the following steps:
 
 #. Any community member may nominate someone or themselves for Committee membership by contacting one of the :ref:`current Committee members <steering_members>`) or by sending an email to ``ansible-community@redhat.com``.
```

Comment:
What if a Committee member wants to nominate someone else?

Reply:
Good point, changed, PTAL